architecture/core_concepts/routes.adoc

```diff
@@ -1062,3 +1063,69 @@ The routers do not clear the `route status` field. To remove the stale entries
 in the route status, use the
 link:https://github.com/openshift/origin/blob/master/images/router/clear-route-status.sh[clear-route-status
 script].
+
+
+[[disable-namespace-ownership-check]]
+== Disabling Namespace Ownership Check
+
+By default, different paths to a host name (and subdomains for wildcard routes)
+can only be claimed by routes in the same namespace as the owning route.
+
+For example, if namespace ns1 creates the oldest route r1 *_www.abc.xyz_*, then it
+owns that host name and subdomain (abc.xyz) for wildcard routes.  If another
+namespace ns2 tries to create a route with say a different path *_www.abc.xyz/path1/path2_*,
+it would fail because a route in another namespace (ns1 in this case) "owns"
+that host.
+
+It becomes more complex with
+xref:../../install_config/router/default_haproxy_router.adoc#using-wildcard-routes[wildcard routes]
+as we now need to look at routes that are granted claims to hosts (*.abc.xyz)
```

Comment:
Can you clarify this?  I think you are saying that the *.foo.com will prevent a claim to x.foo.com from another namespace.